concourse/pipelines/debian-worker-image-build.yaml

```diff
@@ -184,6 +194,106 @@ jobs:
       job: "build-debian-11-worker-arm64"
       result_state: "failure"
       start_timestamp: ((.:start-timestamp-ms))
+- name: build-debian-12-worker
+  plan:
+  - get: compute-image-tools
+  - get: guest-test-infra
+  - task: generate-timestamp
+    file: guest-test-infra/concourse/tasks/generate-timestamp.yaml
+  - load_var: start-timestamp-ms
+    file: timestamp/timestamp-ms
+  - task: generate-id
+    file: guest-test-infra/concourse/tasks/generate-id.yaml
+  - load_var: id
+    file: generate-id/id
+  - task: generate-build-id
+    file: guest-test-infra/concourse/tasks/generate-build-id.yaml
+    vars:
+      id: '((.:id))'
+      prefix: "debian-12-worker"
+  - put: debian-12-worker-gcs
+    params:
+      file: build-id-dir/debian-12-worker*
+    get_params:
+      skip_download: "true"
+  - load_var: gcs-url
+    file: debian-11-worker-gcs/url
```

Comment:
is this correct? `debian-11-worker-gcs` ?

Reply:
Oops- fixed.